src/node/node_state.h

```diff
@@ -1641,33 +1602,41 @@ namespace ccf
       open_frontend(ccf::ActorsType::users, &network.identity->cert);
     }
 
-    std::vector<uint8_t> serialize_create_request(const QuoteInfo& quote_info)
+    std::vector<uint8_t> serialize_create_request(bool recovery = false)
     {
       CreateNetworkNodeToNode::In create_params;
 
-      for (const auto& m_info : config.genesis.members_info)
+      if (!recovery)
```

Comment:
Although this is used/not used for recovery, what it does is really "define_consortium" or "create_consortium", and ought to be named as a flag that way imo. Then in recovery, "create_consortium = false" because we inherit the consortium from the ledger instead.